platforms/Windows/shared/swift.en-us.wxl

```diff
@@ -25,6 +29,8 @@
   <String Id="Utl_ProductName_arm64" Value="Swift Windows Utilities (ARM64)" />
   <String Id="Utl_ProductName_amd64" Value="Swift Windows Utilities (AMD64)" />
   <String Id="Utl_ProductName_x86" Value="Swift Windows Utilities (X86)" />
+  <String Id="Asserts_Toolchain_ProductName" Value="Swift Toolchain (Asserts enabled)" />
+  <String Id="NoAsserts_Toolchain_ProductName" Value="Swift Toolchain (Asserts disabled)" />
```

Comment:
I think that we should just name the variant spelling here ...

```suggestion
  <String Id="Asserts_Toolchain_ProductName" Value="Swift Toolchain (Asserts)" />
  <String Id="NoAsserts_Toolchain_ProductName" Value="Swift Toolchain (NoAsserts)" />
```

I'm thinking about the next variant that would be useful to add: `Asserts+UBSan`, where `Asserts enabled, Undefined Behaviour Sanitizer enabled` seems like it would be pretty long.

Reply:
changed